apps/backend/src/routes/signup/onboard/+page.svelte

```diff
@@ -0,0 +1,87 @@
+<!-- JavaScript code for Continue button -->
+<script lang="ts">
+  let confirm: boolean = false;
+</script>
+
+<!-- Onboard Form - Logo -->
+<div class="skeleton w-32 h-32"></div>
+
+<div class="flex flex-row items-center justify-center h-screen">
+  <!-- Onboard Form - Card -->
+  <div class="w-full max-w-2xl p-10 border border-secondary rounded-md shadow-lg">
+    <!-- Onboard Form - Header -->
+    <h1 class="text-3xl font-semibold text-center mb-5">Get Onboard</h1>
+
+    <h2 class="text-center">Complete your profile to start using Armadillo!</h2>
+
+    <form action="">
+      <!-- Onboard Form - Name Input -->
+      <div>
+        <label class="form-control w-full max-w-xl">
+          <div class="label">
+            <span class="label-text">Full Name</span>
```

Comment:
If this is a label, the `label` element must be used

Reply:
Solve the bloody issue already